doc/classes/Environment.xml

```diff
@@ -305,7 +305,7 @@
 			Replace glow blending mode. Replaces all pixels' color by the glow value.
 		</constant>
 		<constant name="TONE_MAPPER_LINEAR" value="0" enum="ToneMapper">
-			Linear tonemapper operator. Reads the linear data and performs an exposure adjustment.
+			Linear tonemapper operator. Reads the linear data and performs an exposure adjustment by clamping to [0.0,1.0].
```

Comment:
Style wise, there should be a space after the comma in `[0.0, 1.0]`. Additionally, the parser will fail here as `[` and `]` are used for BBcode-like tags, so you should enclose it in `[code][0.0, 1.0][/code]`.

Reply:
Yes; thanks; changed and pushed it to the pull request.